src/app.py

```
@@ -42,22 +40,24 @@ def load_chain():
         search_type="mmr",
-        search_kwargs={"k": TOP_K, "fetch_k": FETCH_K}
+        search_kwargs={"k": TOP_K, "fetch_k": FETCH_K},
     )
 
-    # BM25 リトリーバー（キャッシュから復元）
     bm25_retriever = None
     if os.path.exists(DOCS_CACHE_PATH):
+        # docs_cache.pkl は ingest.py がローカルの Obsidian Vault から生成する
+        # 信頼済みローカルファイルのため pickle デシリアライズは安全
         with open(DOCS_CACHE_PATH, "rb") as f:
             all_docs = pickle.load(f)
```

Ruff (0.15.7)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)